helpers/aave/index.ts

```diff
@@ -76,7 +68,7 @@ async function getPoolFees(pool: AaveLendingPoolConfig, options: FetchOptions):
   .flat()
 
   for (const event of events) {
-    const reserveData = await options.api.call({
+    const reserveData = await sdk.api2.abi.call({
```

Comment:
need to pass chain parameter here, else it will run against ethereum (the default chain)